packages/web-react/src/components/Modal/README.md

```diff
@@ -165,12 +166,26 @@ accessible name for the dialog, e.g. using the `aria-label` attribute on
 </Modal>
 ```
 
+### Hidden Close Button
+
+The close button can by hidden by adding `hideCloseButton` prop to `ModalHeader` component.
+
+```jsx
+<Modal id="modal-example">
+  <ModalDialog>
+    <ModalHeader hideCloseButton />
+    <ModalBody>…</ModalBody>
+  </ModalDialog>
+</Modal>
+```
+
 ### API
 
-| Name         | Type        | Default | Required | Description        |
-| ------------ | ----------- | ------- | -------- | ------------------ |
-| `children`   | `ReactNode` | —       | ✕        | Children node      |
-| `closeLabel` | `string`    | —       | ✕        | Close button label |
+| Name              | Type        | Default | Required | Description                    |
+| ----------------- | ----------- | ------- | -------- | ------------------------------ |
+| `children`        | `ReactNode` | —       | ✕        | Children node                  |
+| `closeLabel`      | `string`    | `Close` | ✕        | Close button label             |
+| `hideCloseButton` | `bool`      | `false` | ✕        | Whether close button is hidden |
```

Comment:
```suggestion
| `hideCloseButton` | `bool`      | `false` | ✕        | Whether close button is rendered |
```

I think we do not just hide the butter. The button is not rendered, I think.

Reply:
fixed